src/api/goods.ts

```diff
@@ -1,11 +1,35 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
+type Good = {
+  id: number
+  name: string
+  color: string
+};
+
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => {
+  return getAll()
+    .then(theGoods => {
+      const five = theGoods.filter(good => {
+        return good.id <= 5;
+      });
```

Comment:
You forget about sorting
Load 5 first goods shows 5 first goods after sorting them by name